src/test/java/codeu/model/store/basic/HashtagStoreTest.java

```diff
@@ -0,0 +1,83 @@
+package codeu.model.store.basic;
+
+import static codeu.model.data.ModelDataTestHelpers.assertHashtagEquals;
+
+import codeu.model.data.Hashtag;
+import codeu.model.data.HashtagCreator;
+import codeu.model.data.ModelDataTestHelpers;
+import codeu.model.store.persistence.PersistentStorageAgent;
+import java.util.HashMap;
+import java.util.UUID;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mockito;
+
+public class HashtagStoreTest {
+  private HashtagStore hashtagStore;
+  private PersistentStorageAgent mockPersistentStorageAgent;
+
+  @Before
+  public void setup() {
+    mockPersistentStorageAgent = Mockito.mock(PersistentStorageAgent.class);
+    hashtagStore = HashtagStore.getTestInstance(mockPersistentStorageAgent);
+  }
+
+  @Test
+  public void testSetAndGetHashtags() {
+    final HashMap<String, Hashtag> hashtags = new HashMap<String, Hashtag>();
+
+    final Hashtag hash1 =
+        new ModelDataTestHelpers.TestHashtagBuilder().withContent("Soccer").build();
```

Comment:
If you do `import static codeu.model.data.ModelDataTestHelpers.TestHashtagBuilder` at the top then you can just say `new TestHashtagBuilder()` here.

Reply:
Indeed, fixed!